styleguide/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
All the files under the styleguide are generated right? Do we want to commit them?

Reply:
yeah, downgrade to node 14.5.0 and run npm install. At least, that's what I needed to do. Technically I was already on node 14.5.0, but I mention that from our earlier discussion.

As for your question @wilwade I was thinking we could potentially host the static styleguide as a page you could visit so people don't have to generate it locally and go to it. It'll be available from here. But come to think of it, there's no reason we have to do that since the files can just be generated. 

What do you think?

Reply:
@claireolmstead Things were actually a bit more messed up than I thought. Your most recent commit changed a bunch of stuff and somehow michelangelo got removed. I'll make a commit here soon that fixes everything.

Reply:
Re: hosting the style guide. I think just generating and running local for now is good. If we want to at some point we can host it via github pages or something and generate it at deploy time.